src/main/scala/ch/epfl/bluebrain/nexus/kg/indexing/ResolverIndexer.scala

```diff
@@ -50,37 +46,52 @@ private class ResolverIndexerMapping[F[_]: Timer](resources: Resources[F])(impli
 
 object ResolverIndexer {
 
+  // $COVERAGE-OFF$
   /**
     * Starts the index process for resolvers across all projects in the system.
     *
     * @param resources     the resources operations
     * @param resolverCache the distributed cache
     */
-  // $COVERAGE-OFF$
-  final def start(resources: Resources[Task], resolverCache: ResolverCache[Task])(
+  final def start[F[_]: Timer](resources: Resources[F], resolverCache: ResolverCache[F])(
       implicit
-      projectCache: ProjectCache[Task],
+      projectCache: ProjectCache[F],
       as: ActorSystem,
-      s: Scheduler,
-      config: AppConfig): StreamCoordinator[Task, ProjectionProgress] = {
+      config: AppConfig,
+      F: Effect[F],
+  ): StreamSupervisor[F, ProjectionProgress] = {
 
-    import ch.epfl.bluebrain.nexus.kg.instances.kgErrorMonadError
+    val kgErrorMonadError = ch.epfl.bluebrain.nexus.kg.instances.kgErrorMonadError
 
-    implicit val indexing = config.keyValueStore.indexing
+    implicit val indexing: IndexingConfig = config.keyValueStore.indexing
 
-    val mapper = new ResolverIndexerMapping[Task](resources)
-    SequentialTagIndexer.start(
-      IndexerConfig
-        .builder[Task]
+    val mapper = new ResolverIndexerMapping[F](resources)
+    TagProjection.start(
+      ProjectionConfig
+        .builder[F]
         .name("resolver-indexer")
         .tag(s"type=${nxv.Resolver.value.show}")
         .plugin(config.persistence.queryJournalPlugin)
-        .retry[KgError](indexing.retry.retryStrategy)
+        .retry[KgError](indexing.retry.retryStrategy)(kgErrorMonadError)
         .batch(indexing.batch, indexing.batchTimeout)
         .offset(Volatile)
         .mapping(mapper.apply)
-        .index(_.traverse(resolverCache.put) *> Task.unit)
+        .index(resolverList => resolverList.traverse(resolverCache.put) >> F.unit)
```

Comment:
Why did you make explicit the variable `resolverList` in this case and not in the other cases? I would say that either everywhere or nowhere (StorageIndexer.scala, ViewIndexer.scala, ...)

Reply:
This was an artefact of my attempt to sort out the compilation errors which were caused by the imported monaderror.